src/CliMenuBuilder.php

```diff
@@ -265,6 +265,36 @@ public function setTitleSeparator(string $separator) : self
         return $this;
     }
 
+    public function setBorder(
```

Comment:
I think we're missing the MenuStyle configuring which should be done in `buildStyle()`